test/demo/demo_unit.js

```diff
@@ -96,7 +96,6 @@ describe('Demo', () => {
       const configPrimitives = new Set(['number', 'string', 'boolean']);
       const exceptions = new Set()
           .add('preferredVariantRole')
-          .add('preferredTextRole')
```

Comment:
Question: why do we need it here?

Reply:
We _did_ need it (past tense) because this config was not available in the demo.  This is an list of exceptions to the "every config must be in the demo" rule.